### PR TITLE
Docker gui

### PR DESCRIPTION
**Pull request by badb**

### CODE REVIEW

**Comment by chfast** [comment]
You should resist putting trivial renames and new features in a single PR. It makes the review a nightmare.

**Comment by badb** [comment]
There aren't any new features - only changes which allows Blender and Luxtask defined in GUI to run as docker files. 
Sorry for the "id" rename, I just couldn't resist :(

File: gnr/task/scripts/docker_blendertask.py [hunk 1]
```diff
@@ -17,21 +17,21 @@ def exec_cmd(cmd):
...
-                              start_task, engine, frame):
+                              start_task, engine, frame, output_format):
     cmd = [
         "{}".format(BLENDER_COMMAND),
         "-b", "{}".format(scene_file),
         "-P", "{}".format(script_file),
         "-o", "{}/{}{}".format(OUTPUT_DIR, outfilebasename, start_task),
         "-E", "{}".format(engine),
-        "-F", "EXR",
+        "-F",  str(output_format).upper(),
```
Comment (chfast) [inline gnr/task/scripts/docker_blendertask.py]: Is there a chance it is not a string?

File: golem/docker/task_thread.py [hunk 2]
```diff
@@ -52,6 +54,9 @@ def run(self):
             with DockerJob(self.image, self.src_code, self.extra_data,
                            self.res_path, work_dir, output_dir) as job:
                 self.job = job
+                if self.check_mem:
+                    mc = MemoryChecker()
+                    mc.start()
```
Comment (chfast) [inline golem/docker/task_thread.py:59]: This is not tested.

**Pull request merged**